test/functional/zmq_test.py

```diff
@@ -6,12 +6,23 @@
 import configparser
 import os
 import struct
+import time
 
 from test_framework.test_framework import BitcoinTestFramework, SkipTest
 from test_framework.util import (assert_equal,
                                  bytes_to_hex_str,
                                  )
 
+def wait_for_multipart(socket, timeout=60):
+    while True:
```

Comment:
Yes, that's better. I've made the change in #10555, and will rebase this PR off that once you've finished your review.